File: mobi_client/mobi_client/Assets/scripts/Config/ConfigModel/MazeConfigModel.cs
namespace ConfigSystem
{
﻿    public class MazeConfigModel :ConfigModel {
        public string name;//名称
        public string scene;//属于场景
        public System.Collections.Generic.List<int> openType;//开启类型
        public System.Collections.Generic.List<int> openParam;//开启参数
        public string jsonName;//迷宫名字
        public System.Collections.Generic.List<int> questId;//包含的支线id
        public System.Collections.Generic.List<System.Collections.Generic.List<int>> reward;//进度奖励
        public string resource;//相关资源路径
    }
}

File: mobi_client/mobi_client/mobi_config/excel_check/temporary/java/MazeConfigModel.java
package com.mobi.config.configModel;
import com.mobi.config.ConfigModel;
import com.mobi.config.wrappedArrayList.*;
public class MazeConfigModel extends ConfigModel {
	public String name;//名称
	public String scene;//属于场景
	public ArrInt openType;//开启类型
	public ArrInt openParam;//开启参数
	public String jsonName;//迷宫名字
	public ArrInt questId;//包含的支线id
	public ArrArrInt reward;//进度奖励
	public String resource;//相关资源路径
}